scripts/requirements.txt

```diff
@@ -4,3 +4,4 @@ intelhex
 recommonmark==0.4.0
 sphinxcontrib-mscgen
 west>=0.7.2
+pylint
```

Comment:
why is this a requirement for all users, and not just ci ?

Reply:
req-ci.txt is generated from this file.

Reply:
got it.